entrypoint.sh

```diff
@@ -45,6 +47,10 @@ cat > /etc/nginx/conf.d/default.conf <<EOF
         location /${OPENSDS_ORCHESTRATION_API_VERSION}/ {
             proxy_pass ${OPENSDS_ORCHESTRATION_URL}/${OPENSDS_HOTPOT_API_VERSION}/;
         }
+
+         location /${SODA_DELFIN_API_VERSION}/ {
+            proxy_pass ${SODA_DELFIN_URL}/${OPENSDS_GELATO_API_VERSION}/;
```

Comment:
please help to understand , why GELATO version here?

Reply:
Since Delfin is using V1 just reused the Gelato version. This will be changed in final PR